react/Autosuggest/Autosuggest.js

```diff
@@ -5,9 +5,16 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import ReactAutosuggest from 'react-autosuggest';
 import IsolatedScroll from 'react-isolated-scroll';
-import omit from 'lodash.omit';
+
+import get from 'lodash/get';
+import invoke from 'lodash/invoke';
+import omit from 'lodash/omit';
```

Comment:
Right, but because it's not explicitly defined in the dependencies in `package.json`, this only works by accident right now.